#include <exception>
using namespace std;

class nowy_u¿ytkownik;
class system_;
class rejestracja;

class rejestracja
{
public: nowy_u¿ytkownik* _unnamed_nowy_u¿ytkownik_;
public: system_* _unnamed_system__;

public: void sprawdzenie_has³a();

public: void utworzenie_karty();

public: void utworzenie_has³a();

public: void uzupe³nienie_danych();

public: rejestracja();

public: void nadanie_id();
};